gcp/modules/locust/main.tf

```diff
@@ -0,0 +1,70 @@
+terraform {
+  backend "gcs" {}
+}
+
+variable "env" {}
+variable "secrets_dir" {}
+variable "charts_dir" {}
+variable "nonce" {}
+
+variable "dns_zones" {
+  type = "map"
+}
+
+resource "null_resource" "locust_link_tasks" {
+  triggers = {
+    nonce = "${var.nonce}"
+  }
+
+  provisioner "local-exec" {
+    command = <<EOF
+      mkdir -p ${var.charts_dir}/locust/tasks
+      for FILE in tasks/*.py; do
+        echo "Creating link for $FILE"
+        ln -sf -T $PWD/$FILE ${var.charts_dir}/locust/$FILE
+      done
+    EOF
+  }
+}
+
+data "template_file" "locust_values" {
+  template = "${file("values.yaml")}"
+
+  vars {
+    locust_workers = "${var.locust_workers}"
+    target_host    = "${var.locust_target_host == "" ?
+      "${var.env == "dev" ? "http" : "https"}://preferences.${
+        substr(var.dns_zones["${var.env}-gcp-gpii-net"], 0,
```

Comment:
We do this calculation in a couple places (flowmanager and preferences values.yaml). Perhaps it's time to move it to Rake-land and pass it through as a variable?

Reply:
Alright then.